sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -744,6 +744,7 @@ object FunctionRegistry {
 
     // comparison operators
     expression[EqualNullSafe]("<=>"),
+    expression[EqualNullSafe]("equal_null", true, Some("3.4.0")),
```

Comment:
We probably need to use `RuntimeReplaceable` for a better func doc. Currently, the func doc is like
`expr1 equal_null expr2 - Returns same result ...` which is quite weird.

Reply:
done